Makefile

```diff
@@ -68,7 +68,7 @@ pipenv-test-install: lib/test-requirements.txt
 	# the Pipfile so we revert these changes after the install.
 	cd lib ; \
 		cp Pipfile Pipfile.bkp ; \
-		pipenv install --dev --skip-lock --sequential -r test-requirements.txt ; \
+		pipenv install --dev --skip-lock --verbose -r test-requirements.txt ; \
```

Comment:
The `--sequential` flag is important for "increasing determinism" in Pipenv (https://pipenv-fork.readthedocs.io/en/latest/advanced.html) - is there a good reason to drop it here?

(We added it while hunting down a dependency heisenbug in the distant past.)

Reply:
I disabled it as an experiment, because the sequential flag, when combined with the underlying behavior of `pip install foo` `pip install bar` not being the same as `pip install foo bar`, means dependencies will almost never be resolved correctly.

Reply:
Seems fragile - `sequential` means "more deterministic", so removing it means "less deterministic". If this is actually the way forward, can we document in a short comment why it's so?

Reply:
It is possible it won't be necessary to keep it, this PR is as much about finding how to make things work as it is to then get those changes merged in. But yes, if it seems necessary to keep it, I'll add a comment.

Reply:
I think the documentation's claim that `sequential` makes it more deterministic is arguable, and I especially think that it achieves whatever determinism it does by very often doing the wrong thing